### PR TITLE
chore: Remove some extra docs

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What and why?

Remove an invalid image tag, and replace it with a link to the official Datadog documentation.

Remove outdated Web support section and extra Data Storage docs (documented on the official documentation site.

refs: #520

### Review checklist

- [ ] This pull request has appropriate unit and / or integration tests 
- [x] This pull request references a Github or JIRA issue

### CODE REVIEW

File: packages/datadog_flutter_plugin/README.md [hunk 1]
```diff
@@ -42,15 +44,8 @@ See [Flutter Web Support](#web_support) for information on current support for F
 
 Use the [Datadog Flutter Plugin][1] to set up Log Management or Real User Monitoring (RUM). The setup instructions may vary based on your decision to use Logs, RUM, or both, but most of the setup steps are consistent.
 
-### Specify application details in the UI
-
-1. In the [Datadog app][2], navigate to **UX Monitoring** > **RUM Applications** > **New Application**.
-2. Choose `Flutter` as the application type.
-3. Provide an application name to generate a unique Datadog application ID and client token.
+For detailed instructions on how to set up the Datadog Flutter Plugin, consult the [Datadog Documentation][11]
```
Comment (0xnm) [inline packages/datadog_flutter_plugin/README.md]:
```suggestion
For detailed instructions on how to set up the Datadog Flutter Plugin, consult the [Datadog Documentation][11].
```

**Review by 0xnm — DISMISSED** [review]

**Review by xgouchet — DISMISSED** [review]

File: packages/datadog_flutter_plugin/README.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real
 Datadog RUM SDK versions < 1.4 support monitoring for Flutter 2.8+.
 Datadog RUM SDK versions >= 1.4 support monitoring for Flutter 3.0+.
 
+For complete documentation, visit Datadog's [official documentation][11].
```
Comment (alai97) [inline packages/datadog_flutter_plugin/README.md]:
```suggestion
For complete documentation, see the [official Datadog documentation][11].
```

File: packages/datadog_flutter_plugin/README.md [hunk 3]
```diff
@@ -42,15 +44,8 @@ See [Flutter Web Support](#web_support) for information on current support for F
 
 Use the [Datadog Flutter Plugin][1] to set up Log Management or Real User Monitoring (RUM). The setup instructions may vary based on your decision to use Logs, RUM, or both, but most of the setup steps are consistent.
 
-### Specify application details in the UI
-
-1. In the [Datadog app][2], navigate to **UX Monitoring** > **RUM Applications** > **New Application**.
-2. Choose `Flutter` as the application type.
-3. Provide an application name to generate a unique Datadog application ID and client token.
+For detailed instructions on how to set up the Datadog Flutter Plugin, consult the [Datadog Documentation][11].
```
Comment (alai97) [inline packages/datadog_flutter_plugin/README.md]:
```suggestion
For instructions on how to set up the Datadog Flutter Plugin, see the [official Datadog documentation][11].
```

**Review by alai97 — DISMISSED** [review]
Looks good, two suggestions!

**Review by xgouchet — APPROVED** [review]

**Pull request merged**